src/get-started/install/_ios-setup.md

```diff
@@ -86,7 +86,35 @@ and an Apple Developer account. If your app is using Flutter plugins,
 you will also need the third-party CocoaPods dependency manager.
 
 <ol markdown="1">
+<li markdown="1">
+<a name="connect"></a>
+To set up physical device deployment in Xcode, 
+connect your device to the USB port on your 
+computer. 
+</li>
+<li markdown="1">
+<a name="wireless"></a>
+[Optional] To leverage wireless debugging, ensure that 
+your device is on the same network as your computer 
+and that the device has a set passcode.  
+
+While the device is attached, open **Xcode > Window > Devices and Simulators**. 
+Select your phone, and check **Connect via Network.**
 
+<img src="/assets/images/docs/setup/connect-wireless.png" alt="Check the box that says 'connect via network' from the devices and simulators page">
+
+Once the network icon appears next to the device name, 
+you can unplug your device from USB. 
+
+Sometimes it takes longer to find network devices. 
+If you don't see your device listed when using `flutter run`, 
+try extending the timeout: `flutter run --device-timeout 10`.
```

Comment:
Does this also need instructions for the IDE plugins?

Reply:
No, the IDEs should not need any custom flags. Since it uses a combination of `xcdevice list` and `xcdevice observe` to determine if the device is connected (from this PR https://github.com/flutter/flutter/pull/123716), it's not limited by the timeout on `xcdevice list` when discovering for the device list in the IDE. And then when it executes a flutter command, it'll wait for the device to connect (from this PR https://github.com/flutter/flutter/pull/122932) if the device is specified from the IDE, so again wouldn't be limited by the timeout on `xcdevice list`.

Reply:
That's what I thought, just checking 🙂